javascript/javascript1/week3/smartPhoneApp.js

```diff
@@ -0,0 +1,141 @@
+// Adding an activity
```

Comment:
It is nice to have the history and context for your decisions, but for the sake of the reviewer, please only submit your final result and avoid keeping commented-out code in. 😄

Reply:
Sorry for that 🥺